app/assets/javascripts/controllers/middleware_datasource/middleware_datasource_controller.js

```diff
@@ -108,7 +127,17 @@ function MwAddDatasourceCtrl($scope, $rootScope, miqService, mwAddDatasourceServ
     vm.step2DsModel.driverClass = dsSelection.driverClass;
 
     mwAddDatasourceService.getExistingJdbcDrivers(serverId).then(function(result) {
-      vm.step2DsModel.existingJdbcDrivers = result;
+      var filteredResult;
+      if(vm.chooseDsModel.xaDatasource){
```

Comment:
will you be fixing this minor styling issue, please? Let me press the green button!